e2e/tests/content-manager/editview.spec.js

```diff
@@ -26,7 +23,7 @@ test.describe('Edit View', () => {
     /**
      * Now we're in the edit view.
      */
-    await page.waitForURL('**/content-manager/collection-types/api::testing.testing/create');
+    await page.waitForURL('**/content-manager/collection-types/api::testing.testing/create?');
```

Comment:
how come you needed to change this? :)

Reply:
I've actually found the test gets stuck here unless we add `?**` to match all query params in the URL